<?xml version="1.0"?>
<launch>

	<arg name="pomdp_model" default="$(find pomdp_ros)/config/agent1_10r.pomdpx" />
	<arg name="pomdp_policy" default="$(find pomdp_ros)/config/agent1_10r.policyx" />
	<arg name="consensus" default="false" />
	<arg name="action_config" default="$(find pomdp_ros)/config/action_time.yaml" />
	<arg name="observation_config" default="$(find pomdp_ros)/config/observation_config.yaml" />
	<arg name="unit_name" default="UAV1" />

	<node name="planner" pkg="pomdp_ros" type="get_next_action.py" output="screen">
		<param name="model" value="$(arg pomdp_model)"/>
		<param name="policy" value="$(arg pomdp_policy)"/>
		<param name="consensus" value="$(arg consensus)"/>
	</node>

	<node name="simulator" pkg="pomdp_ros" type="simulate_action.py" output="screen">
		<param name="action_config" value="$(arg action_config)"/>
	</node>

	<node name="observation_generator" pkg="pomdp_ros" type="observation_generator.py" output="screen">
		<param name="unit_name" value="$(arg unit_name)"/>
		<param name="observation_config" value="$(arg observation_config)"/>
	</node>  

 </launch>
